### ISSUE TITLE
Feedback widget: NVDA does not read the text box name when focus goes to text box.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51896]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
Additional Details:
ScreenReader[NVDA:2018.2],JAWS[18.0.5038],MAS1.3.1
**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Navigate to Status Bar and select "Tweet Feedback" button.
3. Check what NVDA is reading.

**Actual:**
NVDA reads "Comments edit required invalid entry multiline" when focus goes to textbox.
Same issue observed in Jaws also.
**Expected:**

NVDA should read the "Tell us why?(257 characters left)".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The keyboard user will not be able to access the control.
**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference:**
![accessibility a11y_vscode_feedback_screenreader](https://user-images.githubusercontent.com/39204909/41410144-1847e268-6ff6-11e8-8fd1-367ac4d7305b.png)

**Comment by bpasero** [comment]
I think this is fine now, it reads: "Tell us your feedback".
